python-avd/pyavd/_eos_designs/schema/schema_fragments/platform_settings.schema.yml

```diff
@@ -298,6 +298,13 @@ keys:
         reload_delay:
           mlag: 300
           non_mlag: 330
+      - platforms:
```

Comment:
missing management interface if relevant. Also platform_sfe_interface_profile with whatever is supported on this platform.

Reply:
Also p2p_uplinks_mtu probably needs 9194 like the AWEs

Reply:
1. I confirmed with EOS engineering and platform sfe interface profiles are not supported on CloudEOS as this requires physical cores to handle the queue.
2. You are correct, wait_for_convergence is supported
3. Yes p2p_uplinks_mtu max in 9194 on this platform.